Common/include/containers/C2DContainer.hpp

```diff
@@ -621,3 +623,40 @@ using su2activematrix = su2matrix<su2double>;
 
 using su2passivevector = su2vector<passivedouble>;
 using su2passivematrix = su2matrix<passivedouble>;
+
+/*--- Functions to allocate 2D/3D data. ---*/
+
+namespace container_helpers {
+
+/*!
+ * \brief Allocate a vector of varying-size vectors and initialize with some value.
+ * \param[in] M - the first index is >=0 and < M
+ * \param[in] N - the second index is >=0 and < N[first index]
+ * \param[in,out] X - the vector of vectors
```

Comment:
Not on this file please, container_decorators.hpp (sorry I thought I had named it helpers).
Also the general convention we have for the few namespaces in the code is to use CamelCase.

Reply:
Maybe you remember correctly, there is a `namespace container_helpers` also in the top of `C2DContainer.hpp`.

Reply:
Damn... :facepalm: naming really is the hardest problem in programming.
The idea for that container_helpers namespace is to have "private" details (I should have just called it that...) used to define C2DContainer, because some elements of its implementation change when both sizes are dynamic, or when some or all sizes are known at compile time.
Maybe don't use a namespace for the Alloc functions, but give them longer names, like AllocVectorOfVectors.

Reply:
`daad1d0`